packages/core/src/init.ts

```diff
@@ -47,25 +42,20 @@ export function commonInit(userConfiguration: UserConfiguration, buildEnv: Build
   }
 }
 
-export function isValidBrowsingContext(browsingContext: BrowsingContext) {
-  switch (browsingContext) {
-    case BrowsingContext.RUM:
-      if (!areCookiesAuthorized()) {
-        console.error('Cookies are not authorized, RUM will not send any data.')
-        return false
-      }
-      if (isLocalFile()) {
-        console.error('Execution is not allowed in the current context.')
-        return false
-      }
-      return true
-    case BrowsingContext.LOGS:
-      if (isLocalFile()) {
-        console.error('Execution is not allowed in the current context.')
-        return false
-      }
-      return true
+export function isValidCookiesBrowsingContext() {
+  if (!areCookiesAuthorized()) {
+    console.error('Cookies are not authorized, we will not send any data.')
+    return false
+  }
+  return true
+}
+
+export function isValidLocalFileBrowsingContext() {
```

Comment:
valid browsing context naming made sense to regroup protocol and cookie checks, here it seems a bit overkill.

what about something like `checkCookiesAuthorized` and `checkIsNotLocalFile`?